### DISCUSSION TITLE
[RabbitMQ Cluster Operator] Need help setting up Prometheus

### DISCUSSION THREAD

**Discussion by Arun-N** [rabbitmq/rabbitmq-server#5122] [Questions] [answered]
I have a RabbitMQ cluster deployed on Kubernetes using the **kubectl rabbitmq** plugin by following this guide (https://www.rabbitmq.com/kubernetes/operator/quickstart-operator.html)

Prometheus was deployed on same Kubernetes cluster in 'monitoring' namespace using **kube-prometheus-stack helm chart** (https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack)

And then I followed https://www.rabbitmq.com/kubernetes/operator/operator-monitoring.html to connect RabbitMQ to Prometheus.

Prometheus & Grafana works fine, but I'm not able to see any entry for RabbitMQ in Targets section of Prometheus UI like it's mentioned here: 
<img width="951" alt="image" src="https://user-images.githubusercontent.com/20704424/176658603-3677fd9a-cd4f-477e-877c-cb1e65c8a5a4.png">

Targets page is only showing Prometheus related stuff. Am I missing something or doing something wrong?

**Comment by mkuratczyk** [comment, marked as answer]
I see you are now asking questions that suggest you have passed this stage. Is this still an issue?

**Reply by Arun-N** [reply]
yes, this was resolved

**Comment by m0sh1x2** [comment]
Hey, Do you have any info on how you resolved the issue? I am following the exact steps but can't seem to make the operator/prometheus to  scrape the pod or service monitors.

**Reply by Arun-N** [reply]
The guide didn't work for me too. So I cleaned up everything and started from scratch, referencing [this quickstart bash script](https://github.com/rabbitmq/cluster-operator/blob/main/observability/quickstart.sh) instead of the guide. Worked perfectly after that!

**Reply by Arun-N** [reply]
Later on I created my own bash script, tweaked the various options for my use case.

**Reply by m0sh1x2** [reply]
Thank you for the info. I saw the script but didn't even guess that it used the `kube-prometheus-stack`, thought that it used the `prometheus-operator`.
